src/patternfly/components/ToggleGroup/toggle-group.scss

```diff
@@ -0,0 +1,69 @@
+.pf-c-toggle-group {
+  // button
+  --pf-c-toggle-group__button--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-toggle-group__button--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-toggle-group__button--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-toggle-group__button--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-toggle-group__button--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-toggle-group__button--Color: var(--pf-global--Color--100);
+  --pf-c-toggle-group__button--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-toggle-group__button--hover--BackgroundColor: var(--pf-global--disabled-color--200);
+  --pf-c-toggle-group__button--disabled--BackgroundColor: var(--pf-global--disabled-color--200);
+  --pf-c-toggle-group__button--disabled--Color: var(--pf-global--disabled-color--100);
+
+  // Light variation
+  --pf-c-toggle-group__button--m-light--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  // Selected variation
+  --pf-c-toggle-group__button--m-selected--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-toggle-group__button--m-selected--Color: var(--pf-global--Color--light-100);
+
+  // first-last child
+  --pf-c-toggle-group__button--first-child--BorderTopLeftRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-toggle-group__button--first-child--BorderBottomLeftRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-toggle-group__button--last-child--BorderTopRightRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-toggle-group__button--last-child--BorderBottomRightRadius: var(--pf-global--BorderRadius--sm);
+
+  display: flex;
+}
+
+.pf-c-toggle-group__button {
+  padding: var(--pf-c-toggle-group__button--PaddingTop) var(--pf-c-toggle-group__button--PaddingRight) var(--pf-c-toggle-group__button--PaddingBottom) var(--pf-c-toggle-group__button--PaddingLeft);
+  font-size: var(--pf-c-toggle-group__button--FontSize);
+  color: var(--pf-c-toggle-group__button--Color);
+  background-color: var(--pf-c-toggle-group__button--BackgroundColor);
+  border: 0;
+
+  &.pf-m-light {
+    --pf-c-toggle-group__button--BackgroundColor: var(--pf-c-toggle-group__button--m-light--BackgroundColor);
+  }
+
+  &:first-child {
+    border-top-left-radius: var(--pf-c-toggle-group__button--first-child--BorderTopLeftRadius);
+    border-bottom-left-radius: var(--pf-c-toggle-group__button--first-child--BorderBottomLeftRadius);
+  }
+
+  &:last-child {
+    border-top-right-radius: var(--pf-c-toggle-group__button--last-child--BorderTopRightRadius);
+    border-bottom-right-radius: var(--pf-c-toggle-group__button--last-child--BorderBottomRightRadius);
+  }
+
+  &:hover {
```

Comment:
can we add a focus state?

Reply:
added the focus state and going to follow up with @mceledonia on monday